spynnaker/pyNN/models/spike_source/spike_source_array.py

```diff
@@ -167,13 +167,24 @@ def is_recording_spikes(self):
         return self._spike_recorder.record
 
     @overrides(AbstractSpikeRecordable.set_recording_spikes)
-    def set_recording_spikes(self, new_state=True):
+    def set_recording_spikes(
+            self, new_state=True, sampling_interval=None, indexes=None):
+        if sampling_interval is not None:
+            logger.warning("Sampling interval currently not sopported for "
+                           "SpikeSourceArray so being ignored")
+        if indexes is not None:
+            logger.warning("Indexes currently not sopported for "
```

Comment:
sopported -> supported

Reply:
That's what you get for hiring someone who spells like Alan.